### PR TITLE
rewrite links to point to new repository

### PR DESCRIPTION
**Pull request by mikedanese**
DO NOT MERGE UNTIL REPO IS TRANSFERED
#12211

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Please see the [Releases Page](https://github.com/GoogleCloudPlatform/kubernetes/releases)
+Please see the [Releases Page](https://github.com/kubernetes/kubernetes/releases)
```
Comment (thockin) [inline CHANGELOG.md]: http://releases.k8s.io

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ Please fill out either the individual or corporate Contributor License Agreement
 
 Follow either of the two links above to access the appropriate CLA and instructions for how to sign and return it. Once we receive it, we'll be able to accept your pull requests.
 
-***NOTE***: Only original source code from you and other people that have signed the CLA can be accepted into the main repository. This policy does not apply to [third_party](https://github.com/GoogleCloudPlatform/kubernetes/tree/master/third_party).
+***NOTE***: Only original source code from you and other people that have signed the CLA can be accepted into the main repository. This policy does not apply to [third_party](https://github.com/kubernetes/kubernetes/tree/master/third_party).
```
Comment (thockin) [inline CONTRIBUTING.md]: http://code.k8s.io/third_party

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Kubernetes
 
-[![GoDoc](https://godoc.org/github.com/GoogleCloudPlatform/kubernetes?status.png)](https://godoc.org/github.com/GoogleCloudPlatform/kubernetes) [![Travis](https://travis-ci.org/GoogleCloudPlatform/kubernetes.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/kubernetes) [![Coverage Status](https://coveralls.io/repos/GoogleCloudPlatform/kubernetes/badge.svg)](https://coveralls.io/r/GoogleCloudPlatform/kubernetes)
+[![GoDoc](https://godoc.org/github.com/kubernetes/kubernetes?status.png)](https://godoc.org/github.com/kubernetes/kubernetes) [![Travis](https://travis-ci.org/kubernetes/kubernetes.svg?branch=master)](https://travis-ci.org/kubernetes/kubernetes) [![Coverage Status](https://coveralls.io/repos/kubernetes/kubernetes/badge.svg)](https://coveralls.io/r/kubernetes/kubernetes)
```
Comment (thockin) [inline README.md]: godoc.org links should be to k8s.io

File: README.md [hunk 4]
```diff
@@ -8,7 +8,7 @@
 
 <hr>
 
-Kubernetes is an open source system for managing [containerized applications](https://github.com/GoogleCloudPlatform/kubernetes/wiki/Why-Kubernetes%3F#why-containers) across multiple hosts,
+Kubernetes is an open source system for managing [containerized applications](https://github.com/kubernetes/kubernetes/wiki/Why-Kubernetes%3F#why-containers) across multiple hosts,
```
Comment (thockin) [inline README.md]: I don't have a wiki redirector, but we could?

File: README.md [hunk 5]
```diff
@@ -26,7 +26,7 @@ Kubernetes builds upon a [decade and a half of experience at Google running prod
 However, initial development was done on GCE and so our instructions and scripts are built around that.  If you make it work on other infrastructure please let us know and contribute instructions/code.
 
 ### Kubernetes is ready for Production!
-With the [1.0.1 release](https://github.com/GoogleCloudPlatform/kubernetes/releases/tag/v1.0.1) Kubernetes is ready to serve your production workloads.
+With the [1.0.1 release](https://github.com/kubernetes/kubernetes/releases/tag/v1.0.1) Kubernetes is ready to serve your production workloads.
```
Comment (thockin) [inline README.md]: http://releases.k8s.io/v1.0.1

File: cluster/juju/bundles/README.md [hunk 6]
```diff
@@ -54,7 +54,7 @@ For further reading on [Juju Quickstart](https://pypi.python.org/pypi/juju-quick
 ### Using the Kubernetes Client
 
 You'll need the Kubernetes command line client,
-[kubectl](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/kubectl.md)
+[kubectl](https://github.com/kubernetes/kubernetes/blob/master/docs/kubectl.md)
```
Comment (thockin) [inline cluster/juju/bundles/README.md]: http://docs.k8s.io/v1.0/kubectl.md

also, broken link : http://docs.k8s.io/v1.0/user-guide/kubectl/kubectl.html

File: cluster/juju/bundles/README.md [hunk 7]
```diff
@@ -74,7 +74,7 @@ You may also `juju ssh kubernetes-master/0` and call kubectl from that
 machine.
 
 See the
-[kubectl documentation](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/kubectl.md)
+[kubectl documentation](https://github.com/kubernetes/kubernetes/blob/master/docs/kubectl.md)
```
Comment (thockin) [inline cluster/juju/bundles/README.md]: I think you get the point - docs links can go through docs.k8s.io, releases.k8s.io for release links and tagged code, code.k8s.io for HEAD code, etc.

File: cluster/juju/charms/trusty/kubernetes-master/README.md [hunk 8]
```diff
@@ -70,8 +70,8 @@ For more information on the recommended bundle deployment, see the
 #### Post Deployment
 
 To interact with the kubernetes environment, either build or
-[download](https://github.com/GoogleCloudPlatform/kubernetes/releases) the
-[kubectl](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/kubectl.md)
+[download](https://github.com/kubernetes/kubernetes/releases) the
+[kubectl](https://github.com/kubernetes/kubernetes/blob/master/docs/kubectl.md)
```
Comment (thockin) [inline cluster/juju/charms/trusty/kubernetes-master/README.md]: actually, links like these should probably be relative: ../../../../../docs/user-guide/kubectl/kubectl.md

File: README.md (hunk 4 above)
Comment (mikedanese) [inline README.md]: We should probably just move the wiki content to github pages.

File: contrib/diurnal/dc.go [hunk 9]
```diff
@@ -206,7 +206,7 @@ func (s *Scaler) Stop() error {
 var (
 	counts     = flag.String("counts", "", "replica counts, must have at least one (csv)")
 	times      = flag.String("times", "", "times to set replica counts relative to UTC following ISO 8601 (csv)")
-	userLabels = flag.String("labels", "", "replication controller labels, syntax should follow https://godoc.org/github.com/GoogleCloudPlatform/kubernetes/pkg/labels#Parse")
+	userLabels = flag.String("labels", "", "replication controller labels, syntax should follow http://https://godoc.org/k8s.io/kubernetes/pkg/labels#Parse")
```
Comment (eparis) [inline contrib/diurnal/dc.go]: http://https:// ?

File: README.md [hunk 10]
```diff
@@ -26,7 +26,7 @@ Kubernetes builds upon a [decade and a half of experience at Google running prod
 However, initial development was done on GCE and so our instructions and scripts are built around that.  If you make it work on other infrastructure please let us know and contribute instructions/code.
 
 ### Kubernetes is ready for Production!
-With the [1.0.1 release](https://github.com/GoogleCloudPlatform/kubernetes/releases/tag/v1.0.1) Kubernetes is ready to serve your production workloads.
+With the [1.0.1 release](http://changelog.k8s.io/v1.0.1) Kubernetes is ready to serve your production workloads.
```
Comment (eparis) [inline README.md]: no https://changelog.k8s.io ?  porqué?

File: docs/getting-started-guides/coreos/azure/cloud_config_templates/kubernetes-cluster-main-nodes-template.yml [hunk 11]
```diff
@@ -191,7 +191,7 @@ coreos:
         Documentation=http://kubernetes.io/
         Requires=network-online.target
         [Service]
-        Environment=KUBE_RELEASE_TARBALL=https://github.com/GoogleCloudPlatform/kubernetes/releases/download/v1.0.1/kubernetes.tar.gz
+        Environment=KUBE_RELEASE_TARBALL=https://releases.k8s.io/download/v1.0.1/kubernetes.tar.gz
```
Comment (eparis) [inline docs/getting-started-guides/coreos/azure/cloud_config_templates/kubernetes-cluster-main-nodes-template.yml]: link doesn't work, I don't really know why....

Comment (eparis) [inline docs/getting-started-guides/coreos/azure/cloud_config_templates/kubernetes-cluster-main-nodes-template.yml]: That's why, the redirect adds a /tree/ in there before the /download/

File: docs/getting-started-guides/scratch.md [hunk 12]
```diff
@@ -212,7 +212,7 @@ A Kubernetes binary release includes all the Kubernetes binaries as well as the
 You can use a Kubernetes binary release (recommended) or build your Kubernetes binaries following the instructions in the
 [Developer Documentation](../devel/README.md).  Only using a binary release is covered in this guide.
 
-Download the [latest binary release](https://github.com/GoogleCloudPlatform/kubernetes/releases/latest) and unzip it.
+Download the [latest binary release](https://releases.k8s.io/latest) and unzip it.
```
Comment (eparis) [inline docs/getting-started-guides/scratch.md]: redirects to https://github.com/kubernetes/kubernetes/tree/latest (and thus broken)

File: docs/getting-started-guides/scratch.md [hunk 13]
```diff
@@ -226,7 +226,7 @@ we recommend that you run these as containers, so you need an image to be built.
 You have several choices for Kubernetes images:
 - Use images hosted on Google Container Registry (GCR):
   - e.g `gcr.io/google_containers/hyperkube:$TAG`, where `TAG` is the latest
-    release tag, which can be found on the [latest releases page](https://github.com/GoogleCloudPlatform/kubernetes/releases/latest).
+    release tag, which can be found on the [latest releases page](https://releases.k8s.io/latest).
```
Comment (eparis) [inline docs/getting-started-guides/scratch.md]: same broken.

File: docs/getting-started-guides/ubuntu-calico.md [hunk 14]
```diff
@@ -91,7 +91,7 @@ sudo mv -f network-environment /etc
 
 ```
 # Get the Kubernetes Source
-wget https://github.com/GoogleCloudPlatform/kubernetes/releases/download/v0.20.2/kubernetes.tar.gz
+wget https://releases.k8s.io/download/v0.20.2/kubernetes.tar.gz
```
Comment (eparis) [inline docs/getting-started-guides/ubuntu-calico.md]: I think you get it   :)

File: README.md (hunk 10 above)
Comment (mikedanese) [inline README.md]: @thockin we should apply for wildcard cert

**Comment by mikedanese** [comment]
@eparis thanks for the comments! I'm going to break this into smaller chunks for easier review.

**Pull request closed without merging**